scalajslib/src/mill/scalajslib/ScalaJSModule.scala

```diff
@@ -73,4 +80,41 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
 
 }
 
-trait TestScalaJSModule extends ScalaJSModule with TestModule
\ No newline at end of file
+trait TestScalaJSModule extends ScalaJSModule with TestModule {
+  def scalaJSTestDeps = T {
```

Comment:
Those dependencies are required to override dependencies provided by test library. For instance utest depends on `scalajs-test-interface:0.6.19`, while user want to build project with scala.js 0.6.21. We should provide `scalajs-test-interface:0.6.21`